### DISCUSSION TITLE
Is there a way to automatically have settings applied for each video?

### DISCUSSION THREAD

**Discussion by Zolio** [Tyrrrz/YoutubeDownloader#330] [Q&A]
Hey, just downloaded this program and its absolutely amazing but is there a way to apply one set of settings to every video I download? Because I don't want to keep picking 1080p every time I download a video I'd really like to just put the URL and bam, it'd just download

**Comment by Tyrrrz** [comment]
The last selected format should be persisted. Otherwise, the same combination of quality/format may not always be available for every video.
